tests/test_remaining_examples.py

```diff
@@ -27,4 +27,9 @@ def test_for_missing_examples():
         for example in missing
     }
 
-    assert len(missing) == 0, f"Missing {len(missing)} examples: {json.dumps(missing_examples, indent=2)}"
+    if len(missing) > 0:
+        print("| Example |")
+        print("|---------|")
+        for name, link in missing_examples.items():
+            print(f"| [{name}]({link}) |")
```

Comment:
Do we want to print all of them? That will be a lot of scrolling when inspecting other failing tests

Reply:
This test only runs via `make regenerate-test-data` so generally won't be alongside any other failures

Reply:
Very fair! I retract my comment